### PR TITLE
Change master to main

### PR DESCRIPTION
**Pull request by anbraten**

### CODE REVIEW

File: docs/docs/20-usage/50-environment.md [hunk 1]
```diff
@@ -57,7 +57,7 @@ This is the reference list of all environment variables available to your pipeli
 | `CI_REPO_SCM`                    | repository SCM (git)                                                                         |
 | `CI_REPO_URL`                    | repository web URL                                                                           |
 | `CI_REPO_CLONE_URL`              | repository clone URL                                                                         |
-| `CI_REPO_DEFAULT_BRANCH`         | repository default branch (master)                                                           |
+| `CI_REPO_DEFAULT_BRANCH`         | repository default branch (main)                                                           |
```
Comment (6543) [inline docs/docs/20-usage/50-environment.md:60]: This needs to ve changed in code too

File: docs/docusaurus.config.js [hunk 2]
```diff
@@ -210,18 +210,18 @@ module.exports = {
...
           versions: {
-            'current': {
+            current: {
               label: 'Next',
               banner: 'unreleased',
             },
             '1.0': {
               label: '1.0.x',
             },
-            '0.15': {
```
Comment (6543) [inline docs/docusaurus.config.js:224]: Intentional?

Comment (anbraten) [reply docs/docusaurus.config.js:224]: Prettier did this. Maybe I can add it as check to the CI as well 🤔

File: docs/docs/20-usage/50-environment.md (hunk 1 above)
Comment (anbraten) [reply docs/docs/20-usage/50-environment.md:60]: Isn't it changed in this PR already 🤔

File: docs/docusaurus.config.js (hunk 2 above)
Comment (6543) [reply docs/docusaurus.config.js:224]: well I did add it as `1.0` will be interpreded as float not string witch will crash docusaurus

but the others are fine :laughing: 

in any case didnt we already enforce formating ?

File: docs/docs/20-usage/50-environment.md (hunk 1 above)
Comment (6543) [reply docs/docs/20-usage/50-environment.md:60]: https://github.com/woodpecker-ci/woodpecker/blob/524611cf00ea9cc86e2e548244937ec9b7141b24/server/forge/gitlab/convert.go#L107
https://github.com/woodpecker-ci/woodpecker/blob/524611cf00ea9cc86e2e548244937ec9b7141b24/server/forge/gitlab/convert.go#L56-L58
https://github.com/woodpecker-ci/woodpecker/blob/1b43b0bf20c03e63fb270453d2ffa5afed456fb2/server/forge/github/convert.go#L100-L102
https://github.com/woodpecker-ci/woodpecker/blob/1b43b0bf20c03e63fb270453d2ffa5afed456fb2/server/forge/github/convert.go#L159-L161
https://github.com/woodpecker-ci/woodpecker/blob/1b43b0bf20c03e63fb270453d2ffa5afed456fb2/server/forge/github/convert.go#L26
https://github.com/woodpecker-ci/woodpecker/blob/6d2240b2e6f844f267866a26d45b1faac462788c/server/forge/bitbucket/convert.go#L62

Comment (6543) [reply docs/docs/20-usage/50-environment.md:60]: -> #2057

File: docs/docs/30-administration/10-server-config.md [hunk 3]
```diff
@@ -20,9 +20,9 @@ services:
 +     - WOODPECKER_OPEN=true
 ```
 
-You can **also restrict** registration, by keep registration closed and ...  
-... **adding** new **users manually** via the CLI: `woodpecker-cli user add`, or  
-... allowing specific **admin users** via the `WOODPECKER_ADMIN` setting, or  
+You can **also restrict** registration, by keep registration closed and ...
```
Comment (6543) [inline docs/docs/30-administration/10-server-config.md:23]: removing newlines intentional?

File: docs/docs/92-development/03-ui.md [hunk 4]
```diff
@@ -32,7 +32,7 @@ The following list contains some tools and frameworks used by the Woodpecker UI.
 
 ## Messages and Translations
 
-Woodpecker uses [Vue I18n](https://vue-i18n.intlify.dev/) as translation library.  New translations have to be added to `web/src/assets/locales/en.json`. The English source file will be automatically imported into [Weblate](https://translate.woodpecker-ci.org/) (the translation system used by Woodpecker) where all other languages will be translated by the community based on the English source.  
-You must not provide translations except English in PRs, otherwise weblate could put git into conflicts (when someone has translated in that language file and changes are not into master branch yet)
+Woodpecker uses [Vue I18n](https://vue-i18n.intlify.dev/) as translation library.  New translations have to be added to `web/src/assets/locales/en.json`. The English source file will be automatically imported into [Weblate](https://translate.woodpecker-ci.org/) (the translation system used by Woodpecker) where all other languages will be translated by the community based on the English source.
```
Comment (6543) [inline docs/docs/92-development/03-ui.md:35]: rm `\n` ...

File: docs/docs/30-administration/10-server-config.md (hunk 3 above)
Comment (anbraten) [reply docs/docs/30-administration/10-server-config.md:23]: It was done by prettier. If its not a double new-line its not considered anyways

**Review by 6543 — APPROVED** [review]

**Pull request merged**